helm_deploy/hmpps-probation-estate-api/values.yaml

```diff
@@ -45,3 +45,4 @@ generic-service:
 
 generic-prometheus-alerts:
   targetApplication: hmpps-probation-estate-api
+  alertSeverity: hmpps-tier
```

Comment:
Oh - is there where the alerts are going now? I thought it was `manage-a-workforce-dev`

Reply:
`hmpps-tier` alert severity maps to the slack channel `manage_a_workforce_dev`, the name is a bit misleading